upgrading/extending-daml-models/init_background.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+init()
+{
+    cd /tmp
```

Comment:
what's the reason you're doing this in a temp dir?

Reply:
That's how Bernhard did it for the GSG build and run scenario. I'm guessing that's where Katacoda initially puts all the stuff.